jdaviz/conftest.py

```diff
@@ -260,7 +260,7 @@ def _create_spectrum1d_cube_with_fluxunit(fluxunit=u.Jy, shape=(2, 2, 4), with_u
     wcs_dict = {"CTYPE1": "RA---TAN", "CTYPE2": "DEC--TAN", "CTYPE3": "WAVE-LOG",
                 "CRVAL1": 205, "CRVAL2": 27, "CRVAL3": 4.622e-7,
                 "CDELT1": -0.0001, "CDELT2": 0.0001, "CDELT3": 8e-11,
-                "CRPIX1": 0, "CRPIX2": 0, "CRPIX3": 0,
+                "CRPIX1": 0, "CRPIX2": 0, "CRPIX3": 0, "PIXAR_SR": 10,
```

Comment:
With this update to the test fixture, in [`test_spectral_extraction_with_correct_sum_units`](https://github.com/spacetelescope/jdaviz/pull/3253/files#diff-c1dcb77a486f869933272ebabbd7907502b0fd5235e90df94c73287a8f1372bf), the scale factor is no longer the default value of 1, so we need to update the expected values in the test.